src/util.h

```diff
@@ -223,6 +224,14 @@ inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(
   return OneByteString(isolate, data, N - 1);
 }
 
+template <std::size_t N>
+inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(
+    v8::Isolate* isolate,
+    std::array<char, N> arr) {
```

Comment:
This should be:

```suggestion
    const std::array<char, N>& arr) {
```

Otherwise there’s an unnecessary copy operation involved.

(And I am actually not a huge fan of adding this overload; it’s a bit ambiguous whether the right length is `N` or `N-1` … in our case it’s `N - 1`, but with STL containers, it wouldn’t be odd to have no terminating 0 byte. But it’s definitely okay to keep this for this PR, and we’ll see whether we run into trouble or not.)

Reply:
> it’s a bit ambiguous whether the right length is `N` or `N-1` … in our case it’s `N - 1`, but with STL containers, it wouldn’t be odd to have no terminating 0 byte.

I tend to agree... If it was't for
https://github.com/nodejs/node/blob/c0a94ef3c92a422c44a5c123efae9bdb6476ebda/src/node_os.cc#L261
I'd say my suggestion to use std:array wasn't worth it...